engine/evd.mli

```diff
@@ -59,6 +59,8 @@ sig
   (** Horizontal composition : [compose f1 f2] only keeps parts of [f2] where
       [f1] is set. In particular, [f1] and [f2] must have the same length. *)
 
+  val and_ : t -> t -> t
```

Comment:
If this is the intersection you can call it `inter` without colliding with ocaml keywords.